content/pt/case-studies/chinaunicom/index.html

```diff
@@ -0,0 +1,96 @@
+---
+title: Estudo de caso China Unicom
+
+linkTitle: chinaunicom
+case_study_styles: true
+cid: caseStudies
+css: /css/style_case_studies.css
+logo: chinaunicom_featured_logo.png
+featured: true
+weight: 1
+quote: >
+   O Kubernetes melhorou nossa experiência usando a infraestrutura de nuvem. Atualmente, não há tecnologia alternativa que possa substituí-lo.
+---
+
+<div class="banner1" style="background-image: url('/images/CaseStudy_chinaunicom_banner1.jpg')">
+  <h1> ESTUDO DE CASO:<img src="/images/chinaunicom_logo.png" class="header_logo" style="width:25%;margin-bottom:-1%"><br> <div class="subhead" style="margin-top:1%;line-height:1.4em">China Unicom: Como o Kubernetes alavancou a utilização de recursos, aumentando sua eficiência operacional<br>e menores custos em TI
+
+</div></h1>
+
+</div>
+
+<div class="details">
+    Empresa &nbsp;<b>China Unicom</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Localização &nbsp;<b>Beijing, China</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Indústria &nbsp;<b>Telecom</b>
+</div>
+
+<hr>
+<section class="section1">
+<div class="cols">
+  <div class="col1" style="width:100%"">
+    <h2>Desafio</h2>
+    A China Unicom é uma das três principais operadoras de telecomunicações da China e, para atender seus 300 milhões de usuários, a empresa administra vários data centers com milhares de servidores em cada um, usando <a href="https://www.docker.com/">Docker</a> Contêiner e <a href="https://www.vmware.com/">VMWare</a> e <a href="https://www.openstack.org/">OpenStack</a> em sua infra-estrutura desde 2016. Infelizmente, "a taxa de utilização de recursos era relativamente baixa", diz Chengyu Zhang, líder do Grupo de Pesquisa e Desenvolvimento de Tecnologia de Plataforma, "e não tínhamos uma plataforma de nuvem para acomodar nossas centenas de aplicativos". Outrora uma empresa totalmente estatal, a China Unicom, nos últimos anos, obteve investimentos privados da BAT (Baidu, Alibaba, Tencent) e JD.com, e agora está se concentrando no desenvolvimento interno usando tecnologia open source, ao invés de produtos comerciais. Como tal, a equipe do China Unicom Lab de Zhang começou a procurar por um framework de orquestração que fosse de código aberto para sua infraestrutura de nuvem.
```

Comment:
Maybe a comma here will be better 

` usando Docker Contêiner, VMWare e OpenStack`

Reply:
@femrtnz can you review please?

Reply:
perfect